api/handlers.go

```diff
@@ -0,0 +1,111 @@
+package api
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/NebulousLabs/skynet-accounts/user"
+
+	"github.com/NebulousLabs/skynet-accounts/database"
+
+	"github.com/julienschmidt/httprouter"
+	"gitlab.com/NebulousLabs/errors"
+)
+
+// userHandlerGET returns information about an existing user.
+func (api *API) userHandlerGET(w http.ResponseWriter, req *http.Request, ps httprouter.Params) {
+	// TODO Limit to the user themselves.
+	if err := req.ParseForm(); err != nil {
+		WriteError(w, errors.New("Failed to parse parameters."), http.StatusBadRequest)
+	}
+	email := req.Form.Get("email")
+	if email == "" {
+		WriteError(w, errors.New("No email provided."), http.StatusBadRequest)
+		return
+	}
+	users, err := api.DB.UserFindAllByField(req.Context(), "email", email)
```

Comment:
Queries like that should probably be abstracted behind a specific interface method. 

e.g. `user, err := api.staticDB.UserByEmail(email)`

Reply:
Done. `staticDB` is also a good idea.